timer/api/BUILD.bazel

```diff
@@ -0,0 +1,31 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
```

Comment:
Please add ```timer``` into ```build/nogo_config.json``` to enable some linter such as revive.

Reply:
Done